Lib/test/test_random.py

```diff
@@ -263,6 +263,30 @@ def test_gauss(self):
             self.assertEqual(x1, x2)
             self.assertEqual(y1, y2)
 
+    def test_getrandbits(self):
+        # Verify ranges
+        for k in range(0, 1000):
+            self.assertTrue(0 <= self.gen.getrandbits(k) < 2**k)
```

Comment:
While your test is correct, I would prefer a more explicit test:

```suggestion
        for k in range(1, 1000):
            self.assertTrue(0 <= self.gen.getrandbits(k) < 2**k)
        self.assertEqual(self.gen.getrandbits(0), 0)
```